apps/admin/src/pages/sponsors/add-sponsor-form.tsx

```diff
@@ -120,9 +122,20 @@ export function AddSponsorForm() {
           render={({ field }) => (
             <FormItem>
               <FormLabel>Tier</FormLabel>
-              <FormControl>
-                <Input placeholder="Tier" {...field} />
-              </FormControl>
+              <Select onValueChange={field.onChange}>
+                <FormControl>
+                  <SelectTrigger>
+                    <SelectValue placeholder="Select Tier" />
+                  </SelectTrigger>
+                </FormControl>
+                <SelectContent>
+                  {tiers.map((Tier) => (
```

Comment:
Nit: but callback arguments like `Tier`, in this case, should be camelCased.

Reply:
Understood